### PR TITLE
fix recursive read locks, bump version

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: ingest/muxer.go [hunk 1]
```diff
@@ -748,9 +746,7 @@ func (im *IngestMuxer) WriteEntry(e *entry.Entry) error {
 	if e == nil {
 		return nil
 	}
-	im.mtx.RLock()
 	runok := im.state == running
```
Comment (traetox) [inline ingest/muxer.go:749]: don't create this weird temporary variable if we don't need the lock

```
if im.state != running {
    return ErrNotRunning
}

File: ingest/muxer.go [hunk 2]
```diff
@@ -766,9 +762,7 @@ func (im *IngestMuxer) WriteEntryContext(ctx context.Context, e *entry.Entry) er
...
```
Comment (traetox) [inline ingest/muxer.go:765]: same

File: ingest/muxer.go [hunk 3]
```diff
@@ -788,9 +782,7 @@ func (im *IngestMuxer) WriteEntryTimeout(e *entry.Entry, d time.Duration) (err e
 	if e == nil {
 		return
 	}
-	im.mtx.RLock()
 	runok := im.state == running
```
Comment (traetox) [inline ingest/muxer.go:785]: same

File: ingest/muxer.go [hunk 4]
```diff
@@ -1286,7 +1278,6 @@ loop:
 			}
 			continue
 		}
-		im.Info("Connection to %v established, completing negotiation & requesting approval to ingest", tgt.Address)
```
Comment (traetox) [inline ingest/muxer.go:1289]: can we make this a LocalInfo

It helps with debugging.

**Review by traetox — CHANGES_REQUESTED** [review]

File: ingest/muxer.go (hunk 4 above)
Comment (floren) [reply ingest/muxer.go:1289]: he's just moved it lower down

**Review by floren — APPROVED** [review]

**Review by traetox — APPROVED** [review]

**Pull request merged**